Automattic/wp-calypso
# Patterns: Add Patterns to universal header and footer navigation

This pull request addresses https://github.com/Automattic/dotcom-forge/issues/6373 by adding a `WordPress Patterns` item to the `Features` section in the header and footer navigation (just after `WordPress Plugins`):

| Header | Footer
| -----  | -----
| ![image](https://github.com/Automattic/wp-calypso/assets/594356/60b54667-e5c4-462b-b4df-5606eee56be1) | ![image](https://github.com/Automattic/wp-calypso/assets/594356/f8399b6b-6351-4542-9e1a-76a28eeefa7e)

It also fixes inconsistencies and strings that were not correctly localized.

#### Testing instructions

1. Run `git checkout add/patterns-to-header-and-footer` and start your server, or access a [live branch](https://github.com/Automattic/wp-calypso/pull/89122#issuecomment-2032517498)
2. Make sure you are logged out
3. Open the [`Themes` page](http://calypso.localhost:3000/themes)
4. Assert that there is a `WordPress Patterns` link in the `Features` menu in the header
5. Assert that there is a `WordPress Patterns` link in the `Features` section in the footer
6. Log into WordPress.com now
7. Open the [`Patterns` page](http://calypso.localhost:3000/patterns)
8. Repeat steps `#4` and `#5`
